src/main/kotlin/car/business/Cars.kt

```diff
@@ -0,0 +1,33 @@
+package car.business
+
+import kotlin.random.Random
+import kotlin.streams.toList
+
+class Cars(amount: Int, random: Random = Random) {
+    private val RANDOM_MAX_SIZE = 10
+
+    private val cars: List<Car>
+    private val random: Random
+
+    init {
+        cars = ArrayList(amount)
+        for (i in 0 until amount) {
+            cars.add(Car())
+        }
+        this.random = random
+    }
+
+    fun move(): Int {
+        return cars.stream()
+            .map { it.move(random.nextInt(RANDOM_MAX_SIZE)) }
+            .filter { it }
+            .count()
+            .toInt()
```

Comment:
코틀린 컬렉션의 함수를 사용하면 어떨까요?

```suggestion
        return cars.count { it.move(random.nextInt(RANDOM_MAX_SIZE)) }
```